packages/plugins/plugin-table/src/translations.ts

```diff
@@ -2,12 +2,14 @@
 // Copyright 2023 DXOS.org
 //
 
+import { type Resource } from 'i18next';
+
 import { Type } from '@dxos/echo';
 import { TableType } from '@dxos/react-ui-table';
 
 import { TABLE_PLUGIN } from './meta';
 
-export default [
+export const translations: Resource[] = [
```

Comment:
Change to `satisfies Resource[]`, import `Resource` from `react-ui`.

Reply:
Please move all of these into a separate PR if possible, this is making Board much more difficult to review.

Reply:
done throughout